src/solver/impls/petsc/petsc.cxx

```diff
@@ -376,14 +385,15 @@ int PetscSolver::init(int NOUT, BoutReal TIMESTEP) {
     PetscInt dof = n3Dvars();
 
     // Maximum allowable size of stencil in x is the number of guard cells
-    PetscInt stencil_width = mesh->xstart;
+    PetscInt stencil_width_estimate;
+    OPTION(options, stencil_width_estimate, bout::globals::mesh->xstart);
```

Comment:
I know the rest of the file uses the old style, but could this be

```
PetscInt stencil_width_estimate = options["stencil_width_estimate"].withDefault(bout::globals::mesh->xstart);
```

It's nice to immediately initialise variables wherever possible